libc/getpw.c

```diff
@@ -147,5 +159,9 @@ TEST_GROUP_RUNNER(getpwd)
 	RUN_TEST_CASE(getpwd, getpwuid_getroot);
 	RUN_TEST_CASE(getpwd, getpwnam_getnull);
 	RUN_TEST_CASE(getpwd, getpwuid_getnull);
+	/* we can't rename/delete the /etc/passwd file on host */
+#ifndef __linux__
 	RUN_TEST_CASE(getpwd, getpwnam_nopasswdfile);
+	RUN_TEST_CASE(getpwd, getpwuid_nopasswdfile);
+#endif
```

Comment:
imho if we ammend test only for our platform lets do it only for it, so instead og `#ifndef __linux__` lets use `#ifdef __phoenix__`